src/core/tests/library/RuntimeCompositor.py

```diff
@@ -98,3 +98,8 @@ def start_reboot(self, message="Test initiated reboot mock"):
     def mock_sleep(self, seconds):
         pass
 
+    @staticmethod
+    def write_image_default_patch_mode_file(path, data):
+        with open(path, "w+") as file_handle:
+            file_handle.write(data)
```

Comment:
This is a generic function that writes some data to a file. While you may only be using it a specific way, is this the appropriate name for the function given how it's fully generic? Given what it does, does the function need to exist?

Reply:
This is used in a lot of tests in TestAptitudePackagerManager.py and would be used in the other package manager tests once disable auto OS updates is added for those, which was the reason to add this function here. Updated the name to a more generic one